test/framework/config.py

```diff
@@ -661,6 +661,14 @@ def test_log_path(self):
         self.assertTrue(regex.match(res), "Pattern '%s' matches '%s'" % (regex.pattern, res))
         self.assertEqual(log_file_format(), 'log.txt')
 
+    def test_get_build_log_path(self):
+        """Test for build_log_path()"""
+        init_config()
+        self.assertEqual(get_build_log_path(), tempfile.gettempdir())
+        build_log_path = os.path.join(os.getcwd(), 'chicken')
```

Comment:
Please use `self.test_prefix` here, to avoid creating a directory named `chicken` somewhere and making people wonder where it comes from (since the relation between `chicken` and EasyBuild isn't that clear).

`self.test_prefix` is a temporary directory unique to each test which gets cleaned automatically...